src/redux/actions/jobpost.ts

```diff
@@ -15,9 +15,10 @@ export const creatingJobPostRequest = (
     payload: jobForm,
   }
 }
-export const registerJobPostSuccess = (): JobSuccessActionType => {
+export const registerJobPostSuccess = (data: any): JobSuccessActionType => {
```

Comment:
I think `jobPost` would be more descriptive than `data`, and try to give a type also. You can `console.log` to know its type.